User-facing homepage update (#899)
nsidc/earthaccess: docs/index.md

> ![](images/earthaccesslogo.png){alt="hexagonal artwork of a pixalated earth with the word earthaccess" width="10%"}

Reviewer: Shoot, this syntax doesn't work.
![image](https://github.com/user-attachments/assets/0014c57d-0d81-4d25-91ac-6260de8b5d4b)

# Welcome to `earthaccess`!

`earthaccess` is a Python library for searching, downloading, and streaming NASA Earth science data.

> Learn more through a recent  [poster](https://docs.google.com/presentation/d/e/2PACX-1vT8BN1C4FkHrid_eVfOwJjZzKGn9Nomb6mnDj3Nld-nD0ekzoenxQIq6Xzo9VFQsXbixCNskadBwPwM/pub#slide=id.g317ff1cef8c_1_87), [slides](https://docs.google.com/presentation/d/1K5RbQj4OKWt49kznIF9ct-cmWADlvdYA0eI7dA7_fFg/edit#slide=id.g269ad4ab477_0_691) and [video](https://www.youtube.com/watch?v=EIr3j1_wDc0)!

Reviewer: ```suggestion
Learn more through a recent  [poster](https://docs.google.com/presentation/d/e/2PACX-1vT8BN1C4FkHrid_eVfOwJjZzKGn9Nomb6mnDj3Nld-nD0ekzoenxQIq6Xzo9VFQsXbixCNskadBwPwM/pub#slide=id.g317ff1cef8c_1_87), [slides](https://docs.google.com/presentation/d/1K5RbQj4OKWt49kznIF9ct-cmWADlvdYA0eI7dA7_fFg/edit#slide=id.g269ad4ab477_0_691), and [video](https://www.youtube.com/watch?v=EIr3j1_wDc0)!
```

## Get Started

Using your command line, install `earthaccess` from [PyPI](https://pypi.org) using `pip`:

```
python -m pip install earthaccess
```

Or from [conda-forge](https://conda-forge.org) using `mamba` or `conda`:

```
mamba install earthaccess
```

## Why `earthaccess`?
    

> ![*NASA Earthdata using `earthaccess` for impactful science. Art by Allison Horst and Openscapes.*](images/earthaccess_environmental_horst.jpg){alt="3-panel watercolor artwork of environmental data within mountains and streams with arrows labeled 'import' and 'earthaccess' that connect to a data analysis panel and then a communicate panel" width="100%"}

Reviewer: ```suggestion
![*NASA Earthdata using `earthaccess` for impactful science. Art by Allison Horst and Openscapes.*](images/earthaccess_environmental_horst.jpg){alt="3-panel watercolor artwork depicting earthaccess' position as a missing link between environmental data and data analysis. Depicts environmental data within mountains and streams with arrows labeled 'import' and 'earthaccess' that connect to a data analysis panel and then a communicate panel." width="100%"}
```

I felt the alt-text could make the meaning we're trying to convey with the image a bit more up-front.

<br>
    
We can use `earthaccess` to do impactful science, faster:

> - Search Data: Explore NASA's extensive collection of Earth science data.

Reviewer: ```suggestion
- Search Data: Explore NASA's extensive collection of Earth science data. Powered by [python-cmr](https://github.com/nasa/python_cmr).
```

Inspired by recent discussions about how python-cmr's importance as an earthaccess component may be under-communicated.

- Access Data: Download data to your local machine or stream it directly into your Python scripts.

> - Manage Credentials: Easily manage your Earthdata Login and Earthdata Cloud credentials.

Reviewer: ```suggestion
- Authenticate: Manage your Earthdata Login and Earthdata Cloud credentials.
```

## Community

Please join us! Contribute to our open-source community effort:

> - Contributing Guide: Learn how to contribute to `earthaccess`.
> - GitHub Repository: Explore the `earthaccess` codebase.
> - Contact Us: Get in touch with the `earthaccess` team.

Reviewer: ```suggestion
- [Contributing Guide](/contributing/index.md): Learn how to contribute to `earthaccess`.
- [GitHub Repository](https://github.com/nsidc/earthaccess): Explore the `earthaccess` codebase.
- Contact Us: Get in touch with the `earthaccess` team.
```

What should we use for a "contact us" link? GitHub Discussions?
